tests/subsys/bluetooth/mesh/sensor_subsys/src/main.c

```diff
@@ -1234,6 +1234,93 @@ static void apparent_power_check(const struct bt_mesh_sensor_type *sensor_type)
 	}
 }
 
+static void temp_range_check(const struct bt_mesh_sensor_type *sensor_type)
```

Comment:
I guess this function is not required. It is possible to reuse `temp_check` to generalize the channel number. Look at my `voltage_check` and `electric_current_check`.